### ISSUE TITLE
OpcodeName.java:95-99: Get rid of code() method. This...

### ISSUE THREAD

**Issue by 0pdd** [objectionary/jeo-maven-plugin#772]
The puzzle `770-2f429c73` from #770 has to be resolved: 

https://github.com/objectionary/jeo-maven-plugin/blob/9a5a3a64ed2457c09abe76b5b26a932c1019c9ad/src/main/java/org/eolang/jeo/representation/directives/OpcodeName.java#L95-L99

The puzzle was created by h1alexbel on 18-Oct-24. 

Estimate: 35 minutes,  role: DEV. 

If you have any technical questions, don't ask me, submit new tickets instead. The task will be \"done\" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by h1alexbel** [comment]
@volodya-lombrozo I want take this, if you don't mind

**Comment by volodya-lombrozo** [comment]
@h1alexbel Sure

**Comment by volodya-lombrozo** [comment]
@h1alexbel Is this issue blocked by `opeo-maven-plugin`?

**Comment by h1alexbel** [comment]
@volodya-lombrozo yes, this issue depends on https://github.com/objectionary/opeo-maven-plugin/issues/448, which depends on https://github.com/objectionary/opeo-maven-plugin/issues/448

**Comment by volodya-lombrozo** [comment]
@h1alexbel Thanks, I'll take a look

**Comment by 0pdd** [comment]
The puzzle `770-2f429c73` has disappeared from the source code, that's why I closed this issue.

**Comment by 0crat** [comment]
@h1alexbel Great job closing this issue! According to our policy, you've earned +4 points: +8 base, -20 for 272 days delay (max deduction), +16 to meet the minimum reward. While the delay impacted your score, we appreciate your perseverance. Your current balance is [+160](https://www.eolang.org/zerocracy/objectionary-vitals.html). Don't forget to check your [Zerocracy balance](https://www.zerocracy.com/) too! 💪
